parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -315,12 +316,7 @@ installContainerd() {
     else
       exit 25
     fi
-    removeMoby
     removeContainerd
-    retrycmd_no_stats 120 5 25 curl ${MS_APT_REPO}/config/ubuntu/${UBUNTU_RELEASE}/prod.list >/tmp/microsoft-prod.list || exit 25
```

Comment:
moved this duplicated collection of statements (both `installMoby` and `installContainerd` have it) into `installDeps`; in other words, all suppported configurations are going to need these apt config updates, so we should do them in that common place